server/src/main/resources/application.yml

```diff
@@ -135,3 +135,6 @@ mail-jet:
 ---
 github-credentials:
   github_token: ${ GITHUB_TOKEN }
+---
+jackson:
+  serialization-inclusion: ALWAYS
```

Comment:
I'm confused. Isn't this for #2132?

Reply:
we decided to put them together since it is a one line change...probably should have made one story for both of them